src/Bitmovin/api/factories/manifest/DashManifestFactory.php

```diff
@@ -59,6 +63,38 @@ public static function createDashManifestForEncoding(JobContainer $jobContainer,
     }
 
     /**
+     * @param EncodingContainer $encodingContainer
+     * @param DashOutputFormat $dashManifest
+     * @return array
+     */
+    private static function getConfigurationsForEncoding(EncodingContainer $encodingContainer, DashOutputFormat $dashManifest)
```

Comment:
Is the variable naming $dashManifest really correct? Should probably be $dashOutputFormat?